### ISSUE TITLE
Hide icon from Menu Bar

### ISSUE THREAD

**Issue by BetterB0y** [Gaulomatic/AirPodsSanity#2]
I would love to see this feature. I locked my mic already and i don't need to change it now. So its unnecessary icon in menu bar.

**Comment by habovh** [comment]
Good idea, however we'd still need a way to access the settings. An approach similar to what [Velja](https://sindresorhus.com/velja) is doing would be nice: menu bar icon shows for a few seconds when the app is opened, allowing to interact with the menu if needed, then goes away.

**Comment by Gaulomatic** [comment]
This feature has been implemented in 1.0.4. Thanks @habovh for the suggestion on how to implement this. The menubar icon will reappear for 10 seconds when the app is started for again, meaning a seconds instance.

**Comment by Gaulomatic** [comment]
Closed as feature is implemented.

**Comment by Gaulomatic** [comment]
I had to add some more detection methods if the app is re-opened, since the Finder will not really launch the app twice, but instead just activate the already running one.

Now this works when the app is actually instantiated twice (via Terminal) through IPC as well as when the activation event is fired on "re-launching" via Finder, Raycast and such.

I killed version 1.4.0 from the releases and uploaded version 1.4.0.1 in wich everything should work as intended.
